packages/react-combobox/src/components/ComboButton/useComboButton.ts

```diff
@@ -11,18 +14,50 @@ import type { ComboButtonProps, ComboButtonState } from './ComboButton.types';
  * @param props - props from this instance of ComboButton
  * @param ref - reference to root HTMLElement of ComboButton
  */
-export const useComboButton_unstable = (props: ComboButtonProps, ref: React.Ref<HTMLElement>): ComboButtonState => {
+export const useComboButton_unstable = (
+  props: ComboButtonProps,
+  ref: React.Ref<HTMLButtonElement>,
+): ComboButtonState => {
+  const { placeholder, value } = props;
```

Comment:
It also seems that something is wrong with this component. It renders **three** times when an active item changes:

![image](https://user-images.githubusercontent.com/14183168/155345637-7eb3ce49-e345-4a24-9eee-9436ae2c7757.png)

Reply:
fixed 👍